### DISCUSSION TITLE
Applicability to SparkSQL/PySpark/Parquet file types

### DISCUSSION THREAD

**Discussion by zwelshman** [OHDSI/DataQualityDashboard#362] [Q&A] [answered]
Hello, I'm exploring the use of OMOP and this DQ segment on spark clusters and was curious to know if it is compatible with SparkSQL/PySpark/Parquet file types? 

Any pointers to resources would be warmly welcomed!! 

Kind Regards,
Zach

**Comment by katy-sadowski** [comment, marked as answer]
I just saw this, apologies. This package uses [DatabaseConnector](http://ohdsi.github.io/DatabaseConnector/) to connect to a database, and [SqlRender](https://ohdsi.github.io/SqlRender/articles/UsingSqlRender.html) to execute SQL against that database; both do support Spark.  I'm not extremely familiar with the Spark universe so I'd recommend checking the linked docs to see if these tools will work with your specific setup.
